longtask-timing/idlharness.window.js

```diff
@@ -7,8 +7,10 @@
 
 idl_test(
   ['longtasks'],
-  ['performance-timeline'],
-  (idl_array, t) => new Promise((resolve, reject) => {
+  ['performance-timeline', 'hr-time'],
+  (idl_array, t) => new Promise(async (resolve, reject) => {
```

Comment:
```suggestion
  (idl_array, t) => new Promise((resolve, reject) => {
```

I think I've had a linter complain at me for this once, and https://stackoverflow.com/q/43036229 talks about it. Quick demo:
```js
(new Promise(async (resolve, reject) => {
    throw new Error('unhandled');
})).catch(()=>{});
```

I didn't read the spec, but think the return value of the Promise constructor callback is simply ignored. Also, there's no `await` here so it's not needed.